packages/components/src/public/QueryModel/ManageViewsModal.tsx

```diff
@@ -142,7 +142,7 @@ export const ManageViewsModal: FC<Props> = memo(props => {
     const deleteSavedView = useCallback(() => {
         handleAction(async () => {
             const viewName = deleting.name;
-            await api.query.deleteView(schemaQuery, containerPath, viewName, false);
+            await api.query.deleteView(schemaQuery, containerPath, viewName, true);
```

Comment:
They do all seem to. Adding a default value means reordering the parameters (since viewName is also optional). Not sure it's worth it really, so I'm leaving as is.